addons/payment_stripe_sca/models/payment.py

```diff
@@ -52,6 +52,8 @@ def _stripe_request(self, url, data=False, method="POST"):
             "AUTHORIZATION": "Bearer %s" % self.sudo().stripe_secret_key,
             "Stripe-Version": "2019-05-16",  # SetupIntent need a specific version
         }
+        if data and data.get("customer_email"):
+            data["customer_email"] = data["customer_email"].strip()
```

Comment:
If this line addresses a bug with un-stripped email address, it should be fixed in a separate PR IMHO